ansible/library/ptf_portchannel.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+DOCUMENTATION = '''
+module:  ptf_portchannel
+
+short_description: manage portchannel interface in PTF container
+description: start/stop portchannel interface in PTF container with certain configurations
+
+Options:
+    - option-name: cmd
+      description: An action string as [start|stop]
+      required: True
+    - option-name: portchannel_config
+      description: A dict to indicate the portchannel configuration. E.G. {"PortChannel101": { "intfs": [0, 4] } }
+      required: True
+'''
+
+
+EXAMPLES = '''
+- name: Start PTF portchannel
+  ptf_portchannel:
+    cmd: "start"
+    portchannel_config: "{{ portchannel_config }}"
+'''
+
+
+import os
+import re
+import traceback
+
+import jinja2
+from ansible.module_utils.basic import *
```

Comment:
>* [](start = 39, length = 1)

Could you list the units you are importing? Or just `from ansible.module_utils.basic` ?

Reply:
I found too many other files that use this statement to import the ansible module, So I just copy it.